src/transformers/configuration_utils.py

```diff
@@ -249,6 +251,16 @@ def __init__(self, **kwargs):
         # task specific arguments
         self.task_specific_params = kwargs.pop("task_specific_params", None)
 
+        # regression / multi-label classification
+        self.problem_type = kwargs.pop("problem_type", None)
+        allowed_problem_types = (None, "regression", "single_label_classification", "multi_label_classification")
+        if self.problem_type not in allowed_problem_types:
```

Comment:
```suggestion
        allowed_problem_types = ("regression", "single_label_classification", "multi_label_classification")
        if self.problem_type is not None and self.problem_type not in allowed_problem_types:
```
Absolute nit, but I would keep the None separate.